components/slider/index.css

```diff
@@ -455,17 +459,250 @@ governing permissions and limitations under the License.
+
+    @media (forced-colors: active) {
+      background-color: ButtonText;
+    }
+  }
+}
+
+
+.spectrum-Slider-labelContainer {
+  color: var(--highcontrast-slider-label-text-color, var(--mod-slider-label-text-color, var(--spectrum-slider-label-text-color)));
+}
+
+
+.spectrum-Slider--filled {
+  .spectrum-Slider-track:first-child {
+    &::before {
+      background: var(--highcontrast-slider-track-fill-color, var(--mod-slider-track-fill-color, var(--spectrum-slider-track-fill-color)));
+      @media (forced-colors: active) {
+        background: ButtonText;
+      }
+    }
+  }
+}
+
+.spectrum-Slider-fill {
+  &::before {
+    background: var(--highcontrast-slider-track-fill-color, var(--mod-slider-track-fill-color, var(--spectrum-slider-track-fill-color)));
+    @media (forced-colors: active) {
+      background: ButtonText;
+    }
+  }
+}
+
+.spectrum-Slider-ramp {
+  path {
+    fill: var(--highcontrast-slider-ramp-track-color, var(--mod-slider-ramp-track-color, var(--spectrum-slider-ramp-track-color)));
+
+    @media (forced-colors: active) {
+      fill: ButtonText;
+    }
+  }
+}
+
+.spectrum-Slider-handle {
+  border-color: var(--highcontrast-slider-handle-border-color, var(--mod-slider-handle-border-color, var(--spectrum-slider-handle-border-color)));
+  background: var(--highcontrast-slider-handle-background-color, var(--mod-slider-handle-background-color, var(--spectrum-slider-handle-background-color)));
+
+  &:hover {
+    border-color: var(--highcontrast-slider-handle-border-color-hover, var(--mod-slider-handle-border-color-hover, var(--spectrum-slider-handle-border-color-hover)));
+  }
+
+  &.is-focused {
+    border-color: var(--highcontrast-slider-handle-border-color-key-focus, var(--mod-slider-handle-border-color-key-focus, var(--spectrum-slider-handle-border-color-key-focus)));
+
+    &:before {
+      box-shadow: 0 0 0 var(--spectrum-focus-indicator-thickness) var(--highcontrast-slider-handle-focus-ring-color-key-focus, var(--mod-slider-handle-focus-ring-color-key-focus, var(--spectrum-slider-handle-focus-ring-color-key-focus)));
+    }
+  }
+
+  &:active,
+  &.is-dragged {
+    border-color: var(--highcontrast-slider-handle-border-color-down, var(--mod-slider-handle-border-color-down, var(--spectrum-slider-handle-border-color-down)));
+  }
+}
+
+
+.spectrum-Slider--ramp {
+  .spectrum-Slider-handle {
+    box-shadow: 0 0 0 var(--spectrum-slider-handle-gap) var(--highcontrast-slider-ramp-handle-border-color-active, var(--mod-sectrum-slider-ramp-handle-border-color-active, var(--spectrum-slider-ramp-handle-border-color-active)));
+
+    /* currently in tokens v5.0.0 the background-color for the handle are not yet defined */
+    background: var(--mod-slider-ramp-handle-background-color, var(--spectrum-slider-ramp-handle-background-color));
+
+    @media (forced-colors: active) {
+      /* forced-color-adjust required to ensure the "circle" around the handle is transparent */
```

Comment:
@lunarfusion I think I found one instance where the `forced-color-adjust` is required. 
In this case to show the "transparent border" around the Ramps's Handle.

**Without**
Emulated:
<img width="431" alt="Screen Shot 2023-01-25 at 16 24 02" src="https://user-images.githubusercontent.com/52184321/214716870-c96b5323-0e92-4d83-abd4-0a70ba21505c.png">
Windows 11 Edge on ARM:

<img width="505" alt="Screen Shot 2023-01-25 at 16 26 55" src="https://user-images.githubusercontent.com/52184321/214716893-fc2f13f8-97ad-4add-93eb-f7b9587f40f7.png">


**Width**
Emulated: 

<img width="381" alt="Screen Shot 2023-01-25 at 16 24 30" src="https://user-images.githubusercontent.com/52184321/214716959-81cf4f39-3eaf-4d74-a88d-75ee90505411.png">


Windows 11 Edge on ARM:
<img width="460" alt="Screen Shot 2023-01-25 at 16 27 20" src="https://user-images.githubusercontent.com/52184321/214717053-b8e3ae6e-1427-44b5-908f-4d9459a42b51.png">